test/infrastructure/docker/main.go

```diff
@@ -163,17 +164,27 @@ func setupChecks(mgr ctrl.Manager) {
 }
 
 func setupReconcilers(ctx context.Context, mgr ctrl.Manager) {
+	// Set our runtime client into the context for later use
+	runtimeClient, err := container.NewDockerClient()
+	if err != nil {
+		setupLog.Error(err, "unable to establish container runtime connection", "controller", "reconciler")
+		os.Exit(1)
+	}
+	ctx = container.RuntimeInto(ctx, runtimeClient)
```

Comment:
```suggestion
```
nit. I think we don't need the runtimeClient in this Context

Reply:
Ah, I think you are right. Will get that removed.